automates/model_assembly/sandbox.py

```diff
@@ -38,8 +38,8 @@ def load_lambda_function(func_str: str) -> Callable:
         raise UnsafeOperationError(f"found in lambda:\n{func_str}")
 
     # Checking for expected lambda expression header
-    if re.match(r"lambda [A-Za-z0-9_,]*:", func_str) is None:
-        raise RuntimeError(f"Unexpected form for lambda:\n{func_str}")
+    # if re.match(r"lambda [A-Za-z0-9_,]*:", func_str) is None:
```

Comment:
@dpdicken why did you comment out this check? I use this to ensure we don't see any code injection or malformed members amongst the arguments of a lambda function.

Reply:
Thanks for catching this, I had commented it out because a test was failing due to the lambda regex match here failing. It was failing even though the lambda looked perfectly fine to match with the regex to me. However, I just uncommented it and it looks like its passing fine now.